tests/unit/inference/test_inference.py

```diff
@@ -269,12 +269,62 @@ def test(
         torch.cuda.synchronize()
         ds_time = time.time() - start
 
-        if task == "text-generation":
-            bs_output = pipe(query, **inf_kwargs)
-
         # These performance tests are only measuring the time for a single
         # inference request, we just want to check that performance isn't terrible
         #assert ds_time <= (bs_time * 1.1)
+        print(bs_output)
+        print(ds_output)
+        assert assert_fn(bs_output, ds_output)
+
+
+@pytest.mark.inference
+@pytest.mark.sequential
+@pytest.mark.parametrize("model_w_task",
+                         [("gpt2",
+                           "text-generation"),
+                          ("EleutherAI/gpt-neox-20b",
+                           "text-generation"),
+                          ("bigscience/bloom-3b",
+                           "text-generation")],
+                         ids=["gpt2",
+                              "gpt-neox",
+                              "bloom"])
```

Comment:
Can we add OPT into the list but just comment it out? will make it easier to turn back on in a future PR that fixes those issues.

Reply:
I don't think OPT models are supported with multi-gpu at the moment